portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro

```diff
@@ -19,6 +19,14 @@ definition {
 	macro addBreakdownByAttribute {
 		Click(locator1 = "ACSegments#ADD_BREAKDOWN_BUTTON");
 
+		if (IsElementPresent(key_buttonName = "${breakdownName}", locator1 = "ACUtils#GENERIC_BUTTON")) {
```

Comment:
Hey @joshitagaki, I think you could check if the cancel button ("X") is present, if it is you could delete it and after create a new breakdown. I think this mode, you couldn't click in add button and after in cancel button. What do you think?

Reply:
See previous comment. Ex: https://github.com/liferay-ac/liferay-portal/blob/f6f8a53b44af286657ca0c61db045e347bde5c55/portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase#L227-L237